e2e/setup/common.go

```diff
@@ -52,141 +88,121 @@ func common(ctx context.Context, config Config, giantSwarmRelease releasev1alpha
 		}
 	}
 
+	// Ensure draughtsman config.
 	{
-		err := installComponentsFromRelease(ctx, config, giantSwarmRelease)
+		err := ensureDraughtsman(ctx, config)
 		if err != nil {
 			return microerror.Mask(err)
 		}
 	}
 
+	// Deploy App Operator v2 for control plane.
 	{
-		err := installNodeOperator(ctx, config)
+		err := installAppOperator(ctx, config, "2.2.0")
```

Comment:
Do these version numbers have any specific meaning or they are just artificial inside the tests?

Reply:
It is the current latest version of those apps.
It's a manual decisioning process when we deploy control planes:

https://intranet.giantswarm.io/docs/support-and-ops/installation-guide-for-giantnetes/draughtsman-installation/#deploying-app-operator-and-chart-operator

Reply:
I was just thinking if those should be somehow grouped together or configurable when we need to update them in the future, but I guess these are also fine for now, especially if this is gonna change with the new test infrastructure.

Reply:
Couldn't we use `installLatestReleaseChartPackage` to install them? Or the `pullLatestChart` function so that we don't have to manually keep updating these hardcoded versions?